src/modules/Library/features/Popups/AddToBuilderPopup/AddToBuilderPopup.tsx

```diff
@@ -163,7 +163,7 @@ export const AddToBuilderPopup = ({
       const { selectedWorkspace: ownerId } = getValues();
       if (!ownerId) return;
       getWorkspaceApplets({
-        params: { ownerId, limit: APPLETS_WITHOUT_LIMIT, flatList: true },
+        params: { ownerId, limit: MAX_LIMIT, flatList: true },
```

Comment:
please add a TODO to discuss more suitable solution with BA: pagination or infiniti scroll.